jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/plainToLink.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2011  JTalks.org Team
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+/**
+ * Link to website in message content displayed as hyperlink automatically
+ */
+function plainToLink() {
+    var urlOutsideTagsRegexp = /(?![^<]*>|[^<>]*<\/)(\b((?:(https?)|ftp|file):\/\/|www\.|ftp\.)[-A-Z0-9+&@#\/%?=~_|!:,.;]*[-A-Z0-9+&@#\/%=~_|])/gi;
+
+    function urlReplacer(str, link, protocol, offset, s) {
+        switch (protocol) {
+            case 'www.':
+                link = 'http://' + link;
+                break;
+            case 'ftp.':
+                link = 'ftp://' + link;
+                break;
+        }
+        return "<a target='_blank' rel='nofollow' href='" + link + "' >" + link + "</a>";
```

Comment:
1. Fixed! The error was in  $('.post-content-body *) which missed text outside the tags.
2. In progress